README.md

```diff
@@ -44,6 +44,31 @@ To run the following examples you will also need these ROS 1 packages:
 * `rostopic`
 * `rqt_image_view`
 
+### Prerequisites for the examples in this file
+
+In order to make the examples below portable between versions of ROS, we've chosen to define two environment variables, `ROS1_INSTALL_PATH` and `ROS2_INSTALL_PATH`. 
+These are defined as the paths to the installation location of their respective ROS versions.
+
+If you installed Noetic in the default location, then the definition of `ROS1_INSTALL_PATH` would likely be `/opt/ros/noetic`.
+
+Building the bridge as described below requires you to build all of ROS 2.
+We assume that you have downloaded it to `~/ros2_galactic`, and that is where you plan on building it.
+In this case, `ROS2_INSTALL_PATH` will be defined as `~/ros2_galactic/install`.
+
+If you've chosen to install either or both versions of ROS somewhere else, you will need adjust the definitions below to match your installation paths.
+
+Because these definitions are used continuously throughout this file, it would be useful to copy the following lines to your `.bashrc` file (if you're using `bash` as your shell).
```

Comment:
```suggestion
Because these definitions are used continuously throughout this page, it is useful to add the following lines to your shell startup file (`~/.bashrc` if you are using `bash`, `~/.zshrc` if you are using `zsh`).
```